.ci/scripts/generate-debug-docs.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -e
+
+gitUrl=$(git remote get-url origin)
+commit=$(git rev-parse HEAD)
+
+echo "Let's generate the details to help to reproduce it locally ..."
+echo ""
+echo ""
+
+echo "# Let's clone the repo"
+echo "git clone ${gitUrl} .apm-its"
+echo "cd .apm-its"
+echo "git checkout ${commit}"
+echo ""
+echo "# Let's run the apm-integration testing for ${NAME}"
+echo "export ELASTIC_STACK_VERSION=${ELASTIC_STACK_VERSION}"
+echo "export BUILD_OPTS=${BUILD_OPTS}"
+echo "./ci/scripts/${NAME}.sh"
+echo ""
+echo "# Let's save the docker log output as files"
+echo "./scripts/docker-get-logs.sh '${NAME}' || echo 0"
```

Comment:
Good point.
That particular comment is similar to the one we run in the Jenkinsfile.

IIRC, the reason for running the `|| true` was to avoid any environmental issues unrelated to the test of the PR. Those docker logs are generated on the fly to help with the debug, but if for whatever reason the command fails we don't want to fail the build. It's only for debugging purposes but nothing else.

Is it a bit more clear the reason for using that silent command?

Reply:
It makes sense that we wouldn't want to fail the build but couldn't we both continue and inform the user of the failure? Something like `echo "./scripts/docker-get-logs.sh '${NAME}' || echo "Failed to get Docker logs"`?

Reply:
Sure thing, I'll add it now

Reply:
Done :)